### ISSUE TITLE
New Ray Release Breaks VLLM API Server

### ISSUE THREAD

**Issue by kenleejr** [vllm-project/vllm#563]
Hi all, thanks for making such a great repo! I noticed that installing vllm from source now installs ray==2.6.1 and this results in the following error when trying to run an api_server:
python3 -m vllm.entrypoints.api_server
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/vllm/vllm/vllm/entrypoints/api_server.py", line 10, in <module>
    from vllm.engine.async_llm_engine import AsyncLLMEngine
  File "/home/vllm/vllm/vllm/engine/async_llm_engine.py", line 7, in <module>
    from vllm.engine.llm_engine import LLMEngine
  File "/home/vllm/vllm/vllm/engine/llm_engine.py", line 9, in <module>
    from vllm.engine.ray_utils import initialize_cluster, ray, RayWorker
  File "/home/vllm/vllm/vllm/engine/ray_utils.py", line 8, in <module>
    from ray.air.util.torch_dist import TorchDistributedWorker
  File "/home/vllm/.local/lib/python3.10/site-packages/ray/air/__init__.py", line 1, in <module>
    from ray.air.checkpoint import Checkpoint
  File "/home/vllm/.local/lib/python3.10/site-packages/ray/air/checkpoint.py", line 22, in <module>
    from ray.air._internal.remote_storage import (
  File "/home/vllm/.local/lib/python3.10/site-packages/ray/air/_internal/remote_storage.py", line 142, in <module>
    _cached_fs: Dict[tuple, Tuple[float, pyarrow.fs.FileSystem]] = {}
AttributeError: 'NoneType' object has no attribute 'fs'

Downgrading to ray==2.5.1 resolves this issue

**Comment by zhuohan123** [comment]
Can you update pyarrow via `pip install -U pyarrow` and try again?

**Comment by ashleykleynhans** [comment]
I got the same error, and installing `pyarrow` resolved that error, but now I am getting this:

```
Traceback (most recent call last):
  File "/workspace/vllm/vllm/test.py", line 2, in <module>
    from vllm import LLM
  File "/workspace/vllm/vllm/vllm/__init__.py", line 4, in <module>
    from vllm.engine.async_llm_engine import AsyncLLMEngine
  File "/workspace/vllm/vllm/vllm/engine/async_llm_engine.py", line 7, in <module>
    from vllm.engine.llm_engine import LLMEngine
  File "/workspace/vllm/vllm/vllm/engine/llm_engine.py", line 9, in <module>
    from vllm.engine.ray_utils import initialize_cluster, ray, RayWorker
ImportError: cannot import name 'RayWorker' from 'vllm.engine.ray_utils' (/workspace/vllm/vllm/vllm/engine/ray_utils.py)
```

**Comment by ashleykleynhans** [comment]
Works fine if I checkout the `v0.1.2`, but `main` branch gives me issues.

**Comment by zhuohan123** [comment]
Should be fixed by the latest ray and vLLM main branch. Please update and try again. Feel free to reopen the issue if there are still errors.

**Comment by BaiMoHan** [comment]
> Should be fixed by the latest ray and vLLM main branch. Please update and try again. Feel free to reopen the issue if there are still errors.

I've just built v0.1.4 by pulling the source code from Github's main branch, but I'm still encountering this issue. I wonder if the fix for this issue was missed in the v0.1.4 main branch?
` pip install -U pyarrow` solved my problems
